docker/prod/nodejs_v2/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY frontend_v2/package.json frontend_v2/yarn.lock /code/
 RUN yarn install
 ENV PATH="/code/node_modules/.bin:$PATH" 
 COPY frontend_v2/ /code
-RUN ng build --prod --configuration=${NODE_ENV}
+RUN ng build --prod
```

Comment:
Isn't this the issue?

https://stackoverflow.com/questions/50333003/could-not-find-module-angular-devkit-build-angular

Reply:
Yeah but all packages should be installed in `yarn install` right? 

Anyway, I thought we should revert this change and get the code upload worker image pushed so that habitat challenge worker can be set up. Currently, it is not even picking up new submissions and running submissions will all fail because of cilium policy being deployed on every worker restart.